src/ert/_c_wrappers/enkf/ensemble_config.py

```diff
@@ -115,6 +104,8 @@ def _str_to_bool(txt: str) -> bool:
     Any text which is not correctly identified as true or false returns False, but
     with a failure message written to the log:
 
+    >>> import sys
```

Comment:
Probably a good idea to change this to a regular test, instead of a doctest.

Reply:
What if we instead simplify the `doctest` to just test that `_str_to_bool("fail")` returns `False`?
I don't think there's much benefit to test that the logger logs.

Reply:
Sure